### DISCUSSION TITLE
Rimba Raya Biodiversity Reserve Project (VCS 674)

### DISCUSSION THREAD

**Discussion by hai-yr** [opentaps/open-climate-investing#37] [Offsets]
"The Rimba Raya Biodiversity Reserve Project, an initiative by InfiniteEARTH, aims to reduce Indonesia’s emissions by preserving some 64,000 hectares of tropical peat swamp forest. This area, rich in biodiversity including the endangered Bornean orangutan, was slated by the Provincial government to be converted into four palm oil estates. Located on the southern coast of Borneo in the province of Central Kalimantan, the project is also designed to protect the integrity of the adjacent world-renowned Tanjung Puting National Park, by creating a physical buffer zone on the full extent of the ~90km eastern border of the park."
https://registry.verra.org/app/projectDetail/VCS/674

Carbon stock assessment: 
"Teams were divided into biomass and peat sub‐teams to collect a number of measurements for the assessment of aboveground and belowground carbon stocks. Over the course of two months, eight permanent transects totaling 16.0 km in length were installed and surveyed throughout the Carbon Accounting Area. On these transects, a total of thirty‐six 250m x 10m aboveground biomass plots were installed and surveyed (4 or 5 per transect representing a total of nine hectares).
Specialized biomass and peat sub‐team were trained in field techniques including transect and plot layout with line and compass, tree diameter, height and canopy measurements with DBH tapes, clinometers, distance measures with a laser range finder, tree volume areas with Basal Area Factor prism and peat depth measurements with a specially designed peat probe. Team leaders were also trained in data recording and survey management. Operating SOPs and experienced field teams will ensure accurate and efficient monitoring during project field surveys."

Project type: Avoided Planned Deforestation
Project start year: 2008
Project crediting start: 2009
Last verification/monitoring report: 2020
Offset vintage: 2016 (may be able to reach out to Patch to check future vintage availability)

**Comment by hai-yr** [comment]
<img width="621" alt="Screen Shot 2022-10-04 at 4 33 50 PM" src="https://user-images.githubusercontent.com/39865976/193949396-feec06e5-0964-4b25-9056-eb15fd43b873.png">
<img width="520" alt="Screen Shot 2022-10-04 at 4 35 17 PM" src="https://user-images.githubusercontent.com/39865976/193949492-f31b0488-a06d-4a6e-8eac-a3e1788f63a3.png">
Leakage monitoring is conducted for five years beyond the date at which deforestation was projected to occur in the baseline (July 2009 ‐ July 2014) in accordance with the methodology. Five main points outline leakage monitoring and are described below:
1. PT BEST operates plantations only in Central Kalimantan
2. All existing PT BEST concessions will be monitored for development and/or expansion
3. Any new PT BEST concession in Indonesia will be monitored
4. Unpermitted plantation expansion will be monitored within PT BEST’s infrastructure
5. The area of activity shifting leakage and carbon impact will be assessed and reported at each verification
<img width="960" alt="Screen Shot 2022-10-06 at 12 34 58 PM" src="https://user-images.githubusercontent.com/39865976/194403096-9f0c8a07-670c-426f-96ed-697a63ff8013.png">
"The objective of the activity shifting leakage monitoring is to track and account for emissions from any
new oil palm development by the identified agent of deforestation, PT BEST, and its subsidiaries. Activity
is defined both as legal activities defined as the allocation of new concession areas to PT BEST and/or
any (potentially illegal) activity outside their allocated boundaries that could constitute activity shifting
leakage.
It is not the responsibility of the Project to determine who is responsible for any activity detected in the
leakage belt, but rather to demonstrate that the activity is not (likely to be) PT BEST."
<img width="1071" alt="Screen Shot 2022-10-06 at 12 43 51 PM" src="https://user-images.githubusercontent.com/39865976/194404572-c00eb72f-1b4b-4436-b108-7db48d3856e9.png">
<img width="584" alt="Screen Shot 2022-10-06 at 12 45 36 PM" src="https://user-images.githubusercontent.com/39865976/194404830-26f3848d-14ef-40c9-92a1-96944de0a0cc.png">

<img width="553" alt="Screen Shot 2022-10-04 at 11 51 52 AM" src="https://user-images.githubusercontent.com/39865976/193902037-74df0886-ba61-4fba-b9ea-d0824e57d502.png">

**Reply by hai-yr** [reply]
@sichen1234 what are your thoughts on the leakage methodology (see maps and text above)? It is a bit unclear to me if/why the company PT BEST's activity only is monitored as part of leakage calculations...

**Comment by hai-yr** [comment]
I will be adding ratings under here for the same criteria as BeZero plus a couple of added criteria:

- Additionality/Baseline 5/5
Commentary:
Financial attractiveness -> there is no clear financial case for protecting the area from conversion to palm oil fields
Baseline area -> the project is within a planned/permitted area for palm oil concessions

- Permanence/risk of reversal: 2.5/5
Commentary:
While the project has made significant efforts to mitigate risk with active patrols and has included buffer pools in the carbon offset calculations, by nature there is risk of reversal. Buffer pools are helpful but not guaranteed to be sufficient, particularly with changing weather patterns due to climate change.
The project has hired rangers to patrol and protect the site from logging, clearing or fire danger. The reserve is also monitored with satellite and aerial imagery to inform conservation. According to the project documents, fire represents the greatest threat to the permanence of the stored carbon. "Best practices in forest and peat fire management will be taken to mitigate the incidence and spread of peat fires within Project Zone." 
InfiniteEARTH will hold an Ecosystem Restoration Concession License over the project zones and area. This license will provide InfiniteEARTH usage rights for a period of up to 60 years with an option to renew for an additional 30 years beyond that.
"Assessing a 20% Risk Buffer resulted in a total deduction of ‐26,221,564 t CO2e. The net baseline emissions are therefore calculated as 1,969,770 t CO2e for year one, 32,528,323 t CO2e for the first ten years and 104,886,254 t CO2e for the 30 year life of the project."

- Leakage X/5
Commentary: Leakage observations were made within the defined leakage belt. Total land use change was not used as leakage -- total new concessions by PTBEST within the leakage belt were counted as leakage.

- Incentives 3/5
Commentary: Not seeing evidence that this has happened but possible risks include: 1. Such projects could reduce the government's willingness to enact legislation to reduce deforestation 2. Possible incentives to threaten/permit lands for concessions just to sell offsets for them

- Over-crediting X/5
Commentary:

- Political environment X/5
Commentary:

+ Transparency/availability of data 5/5
Commentary: There is a high quantity of relevant project documents and frequent monitoring and verification reports.

+ Transition to net zero 1.5/5
Commentary: This type of project is necessary for bringing global emissions down, but as an emissions avoidance project it should not be used towards a company's claim to be net zero, according to the Science Based Targets initiative.

**Comment by sichen1234** [comment]
For this one too, is there a place to buy this credit?

**Reply by hai-yr** [reply]
https://dashboard.patch.io/projects/pro_prod_266c8b2159c5c091cffc5cc8f27897e5 not sure if you have access to this but there are 1000 tonnes available on Patch.io
